03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,68 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <header>
+        <h1>The Daily Planet</h1>
+        <nav>
+            <a href="#">Sports</a>
+            <a href="#">Local</a>
+            <a href="#">Arts & Entertainment</a>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <img src="http://interactives.dallasnews.com/2016/texas-olympians/images/lede-2400.jpg" />
+            <article>
+                <h2>Texas Gold</h2>
+                <h3><em>More than 100 athletes from the Lone Star State will compete in Brazil</em></h3>
+                <h4>By: Jori Epstein</h4>
+                <p>Texas will be well-represented at the 2016 Olympics in Rio de Janeiro, as over 100 athletes with ties across the state compete in Brazil.</p>
+                <p>We’re tracking all athletes with Texas ties as they compete for their respective nations from Aug. 5-21 in Rio. Texas natives, current residents, anyone who went to high school or university in The Lone Star State? You can find them all
+                    here.</p>
+                <p>The schedule below will update as athletes finish events with results, related stories and TV times. The athletes automatically sort alphabetically, but you can also sort by upcoming events — so you can map out your calendar perfectly
+                    to cheer on Texans.</p>
+            </article>
+        </section>
+    </main>
+    <aside>
+        <a href="https://www.insightly.com/"><img src="https://aka-cdn.adtechus.com/images/412/Ad0St1Sz529Sq0V0Id24953244.png" />
```

Comment:
I believe the location you are pulling this image from is not allowing it to show in the browser. Try changing this image out with another one that doesn't get pulled from a CDN.